lib/babbler.rb

```diff
@@ -12,13 +12,11 @@ def babble(seed = nil)
 
       babble_words = []
 
-      Babbler.config.num_adjectives.times do
-        babble_words.push(adjectives[prng.rand(adjectives.length)])
+      Babbler.config.format.gsub(' ', '').split('').each do |f|
+        babble_words.push(send("format_#{f}")[prng.rand(adjectives.length)])
```

Comment:
```ruby
"a b\t\nc".split(/\s+/)
```

There's probably some ask/tell thing going on here but ```format_a``` is vague to me.  I'd rather just have a case or hash to map "a" to :adjectives and leave the method as `adjectives`.